### PR TITLE
[DNM] Update Doc2

### PR DESCRIPTION
**Pull request by FelixYBW**
## What changes were proposed in this pull request?

update the readme

## How was this patch tested?

NA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# GLuten: A Spark Plugin to Offload SQL Engine to Native Library
+*<b>This plugin is still under active development now. and doesn't have a stable release. Welcome to evaluate it. If you encounter any issues or have any suggestions, please submit to out issue list. We'd love to hear your feedback</b>*
```
Comment (zzcclp) [inline README.md:2]: ` development now. and ` -> `development now, and `

**Review by zzcclp — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# GLuten: A Spark Plugin to Offload SQL Engine to Native Library
+*<b>This plugin is still under active development now, and doesn't have a stable release. Welcome to evaluate it. If you encounter any issues or have any suggestions, please submit to out issue list. We'd love to hear your feedback</b>*
```
Comment (carsonwang) [inline README.md:2]: nit: out -> do you mean `our`?

**Comment by carsonwang** [comment]
I listed a few minor typos in the 1st PR. Maybe we can fix them in this PR together.

**Comment by FelixYBW** [comment]
> I listed a few minor typos in the 1st PR. Maybe we can fix them in this PR together.

Done. Thank you for your careful review!

**Pull request merged**